### PR TITLE
fix #660 link to forum from news post headings

### LINKED ISSUE #660: News links on index page do not really behave like links
These links toggle content dynamically on click:
image

But opening them in a new tab/window (e.g. via the context menu or middle mouse click) just opens the same page, without the contents for the entry being expanded.

### PR DESCRIPTION
**Pull request by TiKevin83**
fixes #660 
Points news post header links to the forum post while preserving toggle behavior for basic clicks
Wires script to auto-open dropdowns for any other cases where this is used
Adds back link to author in news post

### CODE REVIEW

**Comment by meshuggahtas** [comment]
Tried to test it locally, but I have zero news article with Demo and Postgres Sample.

File: TASVideos/wwwroot/js/site.js [hunk 1]
```diff
@@ -73,3 +73,8 @@ if (localStorage.getItem("style-dark") !== null) {
 		forceDarkMode();
 	}
 }
+
+if (location.hash != null && location.hash != "") {
```
Comment (adelikat) [inline TASVideos/wwwroot/js/site.js:77]: if (location.hash) is cleaner here

Comment (brunnerh) [reply TASVideos/wwwroot/js/site.js]: I do not know the policy for browser support is here, but if only support for recent browsers is required that could also be simplified using optional chaining (`?.` operator). [(Support Info)](https://caniuse.com/mdn-javascript_operators_optional_chaining)

Comment (TiKevin83) [reply TASVideos/wwwroot/js/site.js]: We're targeting ES6 minimum support, optional chaining is a tad too new, looks like it was part of ES2020. We should probably implement babel+uglify at some point to let us code with newer features while targeting ES6, but at the same time ES6 is pretty clean already

**Pull request merged**